.env

```diff
@@ -0,0 +1 @@
+export GITHUB_TOKEN=cd07c2a05169db85474a774bfd3eefa153d83c90
```

Comment:
it's correct, just that I can see it on GitHub, which means they probably revoked it.